internal/controller/dashboard_controller.go

```diff
@@ -26,7 +26,7 @@ import (
 	"github.com/giantswarm/observability-operator/pkg/grafana"
 	grafanaclient "github.com/giantswarm/observability-operator/pkg/grafana/client"
 
-	"github.com/giantswarm/observability-operator/internal/controller/predicates"
+	"github.com/giantswarm/observability-operator/internal/predicates"
```

Comment:
There's already a lot going on here. Those changes are not needed here and would reduce the scope of this PR by removing them.

Reply:
Moved elsewhere